### PR TITLE
Feature/cdpt 1661 404 updates

### PR DESCRIPTION
**Pull request by EmilyHazlehurst**

### CODE REVIEW

**Review by EarthlingDavey — COMMENTED** [review]
Looks good! If we're removing GTM from the maintenance page, should we also remove it from the other error pages?

**Review by EarthlingDavey — APPROVED** [review]
Looks good ! 😊

File: public/app/themes/justice/error-pages/maintenance.html [hunk 1]
```diff
@@ -70,7 +34,7 @@ <h1 class="o-title o-title--page">Website temporarily unavailable</h1>
     <div class="l-primary"></div>
     <div class="l-secondary c-copyright">
       <div class="c-copyright-notice">
-        <img src="/app/themes/clarity/dist/images/crown_copyright_logo.png" alt="crown copyright logo">
+        <img src="/app/themes/justice/src/img/crown_copyright_logo.png" alt="crown copyright logo">
```
Comment (wilson1000) [inline public/app/themes/justice/error-pages/maintenance.html:37]: Hi @EmilyHazlehurst the stacks will need to reference `dist` - `src` doesn't exist in the file system.

File: public/app/themes/justice/error-pages/maintenance.html [hunk 2]
```diff
@@ -4,52 +4,16 @@
...
     <div class="u-wrapper__stack--left">
-      <a href="/" rel="home">
-        <img src="/app/themes/clarity/dist/images/moj_logo.png" alt="MoJ Logo">
-        <span class="agency-title l-half-section">Ministry of Justice intranet</span>
-      </a>
-    </div>
-
-    <div class="u-wrapper__stack--right">
-      <a href="/agency-switcher" class="c-logo-bar__switch">Switch to other intranet</a>
+      <img src="/app/themes/justice/src/img/moj-logo.gif" alt="MoJ Logo">
```
Comment (wilson1000) [inline public/app/themes/justice/error-pages/maintenance.html:15]: Can we reference `dist`?

**The process is...**
LaravelMix should process the image from `src`, apply compressions if necessary and place the image in the `dist` directory structure for use in production. `src` isn't there 👍 

Screenshot 2024-05-30 at 13 11 28

File: public/app/themes/justice/error-pages/maintenance.html [hunk 3]
```diff
@@ -4,52 +4,16 @@
...
-      j.src =
-        '//www.googletagmanager.com/gtm.js?id=' + i + dl
-      f.parentNode.insertBefore(j, f)
-    })(window, document, 'script', 'dataLayer', 'GTM-P545JM')
-
-  </script>
-  <!-- End Google Tag Manager -->
-
+  <title>Website temporarily unavailable | Justice UK</title>
+  <link rel="stylesheet" href="/app/themes/justice/src/css/global.css"/>
```
Comment (wilson1000) [inline public/app/themes/justice/error-pages/maintenance.html:8]: `src -> dist` mentioned below 😄

**Review by wilson1000 — CHANGES_REQUESTED** [review]
Hi @EmilyHazlehurst - great stuff
I've made some comments regarding how we prepare images for distribution.

Also, can we please put the GTM code back with the correct GTM ID? This code allow us to create error reports and track error pages in Google Analytics.

**Comment by wilson1000** [comment]
We set up error pages in our `server.conf` configuration. Some rules allow access to certain directories when server access is restricted.

Interestingly, 401 isn't in there, but 400 is. 
What should we do about this?

```php
    error_page 400 /app/themes/justice/error-pages/400.html;
    error_page 404 /app/themes/justice/error-pages/404.html;
    error_page 500 /app/themes/justice/error-pages/500.html;
    error_page 503 /app/themes/justice/error-pages/maintenance.html;

    # Empty location blocks to allow access when "/" location
    # sends an HTTP 503 during maintenance mode
    location /app/themes/justice/error-pages/ { }
    location /app/themes/justice/dist/ { }
```

**Comment by wilson1000** [comment]
Good morning @EmilyHazlehurst 👋 

For error pages, I've created a file in `dist/patch/js/ccfw-check-consent.js` that can be used to drop the GTM code. We need to make sure that the GTM ID is available on the page, in a `data-allowed` attribute I think 🤔  - please do check this if you choose to use the file, I hope it's helpful.

https://github.com/ministryofjustice/justice-gov-uk/blob/main/public/app/themes/justice/src/patch/js/ccfw-check-consent.js

**Comment by EmilyHazlehurst** [comment]
> Good morning @EmilyHazlehurst 👋
> 
> For error pages, I've created a file in `dist/patch/js/ccfw-check-consent.js` that can be used to drop the GTM code. We need to make sure that the GTM ID is available on the page, in a `data-allowed` attribute I think 🤔 - please do check this if you choose to use the file, I hope it's helpful.
> 
> https://github.com/ministryofjustice/justice-gov-uk/blob/main/public/app/themes/justice/src/patch/js/ccfw-check-consent.js

Thanks @wilson1000. I think for now it's probably best to keep this as it is as the goal of the ticket was just to change the title and and search link on the pages. I've added the original gtm code back in and have updated the ID.

Perhaps we can improve this further as part of the frontend work, since the error pages have some accessibility issues and are quite messy? I can make a ticket for it today.

**Review by wilson1000 — APPROVED** [review]
Thanks Emily 😄

**Pull request merged**